README.md

```diff
@@ -192,15 +192,15 @@ If you are interested in making enhancement to Forte, please first go over our [
 ### Supported By
 
 <p align="center">
-   <img src="https://asyml.io/assets/institutions/cmu.png", width="200" align="top">
+   <img src="https://raw.githubusercontent.com/asyml/forte/master/docs/_static/img/CMURedWordmark200pxSquare.png", width="200" align="top">
       &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-   <img src="https://www.ucsd.edu/_resources/img/logo_UCSD.png" width="200" align="top">
+   <img src="https://raw.githubusercontent.com/asyml/forte/master/docs/_static/img/UCSanDiegoLogo-BlueSquare.png" width="200" align="top">
```

Comment:
try these links yourself

Reply:
I think those links will be valid after it's merged. I tried to change user and branch names to mine and it's working.

Reply:
Or we can change the hyperlink to temporary valid one and then change it back to master branch links after it's merged.

Reply:
We cannot preview the images this way. We usually directly drag the image to any GitHub editor box and it will be uploaded immediately and you can use that link, like this:

![UCSanDiegoLogo-BlueSquare](https://user-images.githubusercontent.com/1015991/165797595-5809f111-0ce7-458c-ac3f-9e6480711a11.png)

And it won't appear in our repo, which is good since it reduces the repo size.